### ISSUE TITLE
Running `make web-build` does not work for me

### ISSUE THREAD

**Issue by jhiemstrawisc** [PelicanPlatform/pelican#403]
On my Mac, I run:
```
make web-clean

make web-build USE_DOCKER=1
```

This gives me an error:
```
> src@0.1.0 build
> next build

Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

   Creating an optimized production build ...
npm ERR! path /webapp
npm ERR! command failed
npm ERR! signal SIGBUS
npm ERR! command sh -c next build

npm ERR! A complete log of this run can be found in: /root/.npm/_logs/2023-11-21T22_25_32_905Z-debug-0.log
make: *** [origin_ui/src/out/index.html] Error 1

```

@CannonLock, do you have any idea whether this is something I'm doing or if it's a proper bug?

**Comment by CannonLock** [comment]
@jhiemstrawisc Can you drop the full logs?

**Comment by jhiemstrawisc** [comment]
```
DKXD6CXM3G:pelican jhiemstra$ make web-clean
CLEAN origin_ui/src/out origin_ui/src/.next
DKXD6CXM3G:pelican jhiemstra$ make web-build USE_DOCKER=1
go: downloading github.com/jellydator/ttlcache/v3 v3.0.1
go: downloading github.com/prometheus/client_golang v1.16.0
go: downloading github.com/sirupsen/logrus v1.8.1
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/gin-gonic/gin v1.9.1
go: downloading github.com/lestrrat-go/jwx/v2 v2.0.16
go: downloading modernc.org/sqlite v1.25.0
go: downloading github.com/hashicorp/go-version v1.6.0
go: downloading github.com/oschwald/geoip2-golang v1.9.0
go: downloading github.com/spf13/viper v1.16.0
go: downloading golang.org/x/net v0.17.0
go: downloading golang.org/x/oauth2 v0.10.0
go: downloading github.com/tg123/go-htpasswd v1.2.1
go: downloading golang.org/x/crypto v0.14.0
go: downloading golang.org/x/term v0.13.0
go: downloading github.com/cavaliercoder/grab v2.0.0+incompatible
go: downloading github.com/go-ini/ini v1.67.0
go: downloading github.com/golang-jwt/jwt v3.2.2+incompatible
go: downloading kernel.org/pub/linux/libs/security/libcap/cap v1.2.69
go: downloading github.com/alecthomas/units v0.0.0-20211218093645-b94a6e3cc137
go: downloading github.com/go-kit/log v0.2.1
go: downloading github.com/grafana/regexp v0.0.0-20221122212121-6b5c0a4cb7fd
go: downloading github.com/mwitkow/go-conntrack v0.0.0-20190716064945-2f068394615f
go: downloading github.com/oklog/run v1.1.0
go: downloading github.com/prometheus/common v0.44.0
go: downloading github.com/prometheus/prometheus v0.46.0
go: downloading github.com/zsais/go-gin-prometheus v0.1.0
go: downloading go.uber.org/atomic v1.11.0
go: downloading github.com/spf13/cobra v1.7.0
go: downloading github.com/spf13/pflag v1.0.5
go: downloading github.com/youmark/pkcs8 v0.0.0-20201027041543-1326539a0a0a
go: downloading gopkg.in/yaml.v3 v3.0.1
go: downloading github.com/studio-b12/gowebdav v0.9.0
go: downloading github.com/vbauerster/mpb/v8 v8.6.1
go: downloading golang.org/x/sync v0.3.0
go: downloading golang.org/x/sys v0.13.0
go: downloading github.com/gin-contrib/sse v0.1.0
go: downloading github.com/mattn/go-isatty v0.0.19
go: downloading github.com/beorn7/perks v1.0.1
go: downloading github.com/cespare/xxhash/v2 v2.2.0
go: downloading github.com/prometheus/client_model v0.4.0
go: downloading github.com/prometheus/procfs v0.11.0
go: downloading google.golang.org/protobuf v1.31.0
go: downloading github.com/oschwald/maxminddb-golang v1.11.0
go: downloading github.com/lestrrat-go/blackmagic v1.0.2
go: downloading github.com/lestrrat-go/httprc v1.0.4
go: downloading github.com/lestrrat-go/iter v1.0.2
go: downloading github.com/lestrrat-go/option v1.0.1
go: downloading github.com/fsnotify/fsnotify v1.6.0
go: downloading github.com/mitchellh/mapstructure v1.5.0
go: downloading github.com/spf13/afero v1.9.5
go: downloading github.com/spf13/cast v1.5.1
go: downloading github.com/spf13/jwalterweatherman v1.1.0
go: downloading github.com/GehirnInc/crypt v0.0.0-20200316065508-bb7000b8a962
go: downloading kernel.org/pub/linux/libs/security/libcap/psx v1.2.69
go: downloading github.com/go-logfmt/logfmt v0.6.0
go: downloading github.com/jpillora/backoff v1.0.0
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading github.com/julienschmidt/httprouter v1.3.0
go: downloading github.com/prometheus/common/sigv4 v0.1.0
go: downloading github.com/edsrzf/mmap-go v1.1.0
go: downloading github.com/stretchr/testify v1.8.4
go: downloading go.opentelemetry.io/otel v1.18.0
go: downloading go.opentelemetry.io/otel/trace v1.18.0
go: downloading golang.org/x/exp v0.0.0-20230713183714-613f0c0eb8a1
go: downloading github.com/gogo/protobuf v1.3.2
go: downloading github.com/oklog/ulid v1.3.1
go: downloading github.com/golang/snappy v0.0.4
go: downloading github.com/klauspost/compress v1.16.7
go: downloading github.com/prometheus/exporter-toolkit v0.10.0
go: downloading go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.44.0
go: downloading github.com/json-iterator/go v1.1.12
go: downloading github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
go: downloading github.com/acarl005/stripansi v0.0.0-20180116102854-5a71ef0e047d
go: downloading github.com/mattn/go-runewidth v0.0.15
go: downloading github.com/VividCortex/ewma v1.2.0
go: downloading github.com/go-playground/validator/v10 v10.14.0
go: downloading github.com/pelletier/go-toml/v2 v2.0.8
go: downloading github.com/ugorji/go/codec v1.2.11
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.4
go: downloading github.com/lestrrat-go/httpcc v1.0.1
go: downloading github.com/subosito/gotenv v1.4.2
go: downloading github.com/hashicorp/hcl v1.0.0
go: downloading gopkg.in/ini.v1 v1.67.0
go: downloading github.com/magiconair/properties v1.8.7
go: downloading golang.org/x/text v0.13.0
go: downloading github.com/Azure/azure-sdk-for-go/sdk/azcore v1.7.0
go: downloading github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.0
go: downloading github.com/google/uuid v1.3.0
go: downloading go.uber.org/goleak v1.2.1
go: downloading github.com/aws/aws-sdk-go v1.44.302
go: downloading github.com/dennwc/varint v1.0.0
go: downloading github.com/coreos/go-systemd/v22 v22.5.0
go: downloading github.com/go-openapi/strfmt v0.21.7
go: downloading github.com/prometheus/alertmanager v0.25.0
go: downloading github.com/shurcooL/httpfs v0.0.0-20230704072500-f1e31cf0ba5c
go: downloading github.com/felixge/httpsnoop v1.0.3
go: downloading go.opentelemetry.io/otel/metric v1.18.0
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/modern-go/reflect2 v1.0.2
go: downloading github.com/rivo/uniseg v0.4.4
go: downloading github.com/go-logr/logr v1.2.4
go: downloading github.com/gabriel-vasile/mimetype v1.4.2
go: downloading github.com/go-playground/universal-translator v0.18.1
go: downloading github.com/leodido/go-urn v1.2.4
go: downloading github.com/golang/protobuf v1.5.3
go: downloading github.com/Azure/azure-sdk-for-go/sdk/internal v1.3.0
go: downloading github.com/AzureAD/microsoft-authentication-library-for-go v1.0.0
go: downloading github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
go: downloading github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2
go: downloading golang.org/x/time v0.3.0
go: downloading github.com/asaskevich/govalidator v0.0.0-20230301143203-a9d515a09cc2
go: downloading github.com/go-openapi/errors v0.20.4
go: downloading go.mongodb.org/mongo-driver v1.12.0
go: downloading github.com/go-openapi/swag v0.22.4
go: downloading github.com/go-openapi/validate v0.22.1
go: downloading github.com/go-logr/stdr v1.2.2
go: downloading github.com/go-playground/locales v0.14.1
go: downloading github.com/kylelemons/godebug v1.1.0
go: downloading github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
go: downloading github.com/mailru/easyjson v0.7.7
go: downloading github.com/go-openapi/analysis v0.21.4
go: downloading github.com/go-openapi/jsonpointer v0.20.0
go: downloading github.com/go-openapi/loads v0.21.2
go: downloading github.com/go-openapi/spec v0.20.9
go: downloading github.com/golang-jwt/jwt/v4 v4.5.0
go: downloading github.com/josharian/intern v1.0.0
go: downloading github.com/go-openapi/jsonreference v0.20.2
go: downloading modernc.org/libc v1.24.1
go: downloading github.com/jmespath/go-jmespath v0.4.0
go: downloading modernc.org/memory v1.6.0
go: downloading modernc.org/mathutil v1.5.0
go: downloading github.com/dustin/go-humanize v1.0.1
go: downloading github.com/remyoudompheng/bigfft v0.0.0-20230129092748-24d4a6f8daec
[+] Building 0.8s (10/10) FINISHED                                                                                                                                                                                                                                               
 => [internal] load .dockerignore                                                                                                                                                                                                                                           0.0s
 => => transferring context: 122B                                                                                                                                                                                                                                           0.0s
 => [internal] load build definition from Dockerfile                                                                                                                                                                                                                        0.0s
 => => transferring dockerfile: 792B                                                                                                                                                                                                                                        0.0s
 => [internal] load metadata for docker.io/library/node:18-alpine                                                                                                                                                                                                           0.7s
 => [auth] library/node:pull token for registry-1.docker.io                                                                                                                                                                                                                 0.0s
 => [internal] load build context                                                                                                                                                                                                                                           0.0s
 => => transferring context: 34B                                                                                                                                                                                                                                            0.0s
 => [1/4] FROM docker.io/library/node:18-alpine@sha256:3428c2de886bf4378657da6fe86e105573a609c94df1f7d6a70e57d2b51de21f                                                                                                                                                     0.0s
 => CACHED [2/4] COPY package.json package.json                                                                                                                                                                                                                             0.0s
 => CACHED [3/4] RUN npm install && npm i -g next                                                                                                                                                                                                                           0.0s
 => CACHED [4/4] WORKDIR /webapp                                                                                                                                                                                                                                            0.0s
 => exporting to image                                                                                                                                                                                                                                                      0.0s
 => => exporting layers                                                                                                                                                                                                                                                     0.0s
 => => writing image sha256:2250b0d1915346de3d0e6dc5ff36900b83f53192fef7eaf453abb49968ab621a                                                                                                                                                                                0.0s
 => => naming to docker.io/library/origin-ui                                                                                                                                                                                                                                0.0s

> src@0.1.0 build
> next build

Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

   Creating an optimized production build ...
npm ERR! path /webapp
npm ERR! command failed
npm ERR! signal SIGBUS
npm ERR! command sh -c next build

npm ERR! A complete log of this run can be found in: /root/.npm/_logs/2023-11-21T22_25_32_905Z-debug-0.log
make: *** [origin_ui/src/out/index.html] Error 1
```

**Comment by jhiemstrawisc** [comment]
I managed to solve my inability to run `make web-build` by borrowing @turetske's bug-fixing aura. Steps to solve were:
1. I moved back into my dev container (which is different than the dev container we publish, but very similar)
2. I temporarily renamed `/origin_ui/src/package-lock.json` and `/origin_ui/src/node_modules` and re-ran `npm install`
4. I tried re-running `make web-build`, with no success
3. I restored the two previous files to their original names
5. Emma came to look at the errors, so I re-ran to demonstrate. Finally, everything worked.
